docs/en/ingest-management/getting-started.asciidoc

```diff
@@ -162,30 +194,109 @@ The {agent} is currently in Experimental and should not be used in production
 This will replace your current settings. Do you want to continue? [Y/n]:
 ----
 
-After that, run the agent with:
-
+. Run the Agent:
++
 [source,shell]
 ----
 ./elastic-agent run
 ----
 
-In the {ingest-manager} app, click **Continue** to get back to the **{fleet}** tab. This will now show the newly enrolled agent.
-
+. In the {ingest-manager} app, click **Continue** to go to the **{fleet}**
+tab, where you should see the newly enrolled agent.
++
 [role="screenshot"]
 image::images/kibana-ingest-manager-fleet-agents.png[{ingest-manager} app showing enrolled agents]
 
-When you want to unenroll an agent, choose **Unenroll** from the **Actions** menu for this agent. This will invalidate the API key the agent is using to connect to {es}. The {agent} will continue to run, but will not be able to send data, and show this error instead:
+//REVIEWERS: This is super cool, but we kind of gloss over the notion that this
+//only works if the logs are stored in the location expected for the service
+//your monitoring. Do we want to tell users how to override default paths, or
+//is that unnecessary for this release?
```

Comment:
Good point. I didn't leave it out on purpose, I think it would be a useful addition.